includes/Story_Post_Type.php

```diff
@@ -513,6 +515,7 @@ public function get_editor_settings() {
 				'storyId'          => $story_id,
 				'previewLink'      => get_preview_post_link( $story_id, $preview_query_args ),
 				'maxUpload'        => $max_upload_size,
+				'useDemoContent'   => $demo,
```

Comment:
Things named `useFoo` in React are generally expected to be hooks and breaking this assumption can confuse devs (and tooling). Booleans really should be named `isFoo`, `hasFoo`, or `shouldFoo`. So I recommend changing it to `shouldUseDemoContent` or simply `isDemo` throughout the application.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/5507/commits/48833436b29c22caea967b6c7870061bbb1f769e